apps/web/src/pages/templates/components/email-editor/EmailContentCard.tsx

```diff
@@ -28,7 +28,11 @@ export function EmailContentCard({
   const { integrations = [] } = useActiveIntegrations();
   const [integration, setIntegration]: any = useState(null);
 
-  const { enabled } = useIntegrationLimit(ChannelTypeEnum.EMAIL);
+  const {
+    enabled,
+    data: { limit, count },
+  } = useIntegrationLimit(ChannelTypeEnum.EMAIL);
+  const isLimitReached = enabled && limit === count;
```

Comment:
```suggestion
  const {
    enabled: limitEnabled,
    data: { limit, count },
  } = useIntegrationLimit(ChannelTypeEnum.EMAIL);
  const isLimitReached = limitEnabled && limit === count;
```
I was confused as thought this `enabled` was meant for an integration active. But it is to have the limit enabled.